src/sql/plugins/workflow/modules/appointmentants/plugin/create_db_workflow-appointmentants.sql

```diff
@@ -1,10 +1,22 @@
+DROP TABLE IF EXISTS workflow_task_ants_appointment;
+DROP TABLE IF EXISTS workflow_task_ants_appointment_history;
+
 --
 -- Table structure for the ANTS workflow tasks 
 --
-DROP TABLE IF EXISTS workflow_task_ants_appointment;
 CREATE TABLE workflow_task_ants_appointment(
   id_task INT NOT NULL,
   id_form INT DEFAULT NULL,
   id_field_entry INT DEFAULT NULL,
   PRIMARY KEY (id_task)
+);
+
+--
+-- Table structure for the ANTS workflow tasks' history
+--
```

Comment:
Add this instruction DROP TABLE IF EXISTS workflow_task_ants_appointment_history;